api/filters/annotations/annotations_test.go

```diff
@@ -162,85 +146,47 @@ kind: Bar
 metadata:
   name: instance
 `
-	annos := map[string]string{
+	annos := annoMap{
 		"sleater": "kinney",
 		"sonic":   "youth",
 	}
-	config := builtinconfig.MakeDefaultConfig()
 	filter := Filter{Annotations: annos}
-	filter.FsSlice = config.CommonAnnotations
-
-	var out bytes.Buffer
-	rw := kio.ByteReadWriter{
-		Reader: bytes.NewBufferString(input),
-		Writer: &out,
-	}
-
-	err := kio.Pipeline{
-		Inputs:  []kio.Reader{&rw},
-		Filters: []kio.Filter{filter},
-		Outputs: []kio.Writer{&rw},
-	}.Execute()
-	if !assert.NoError(t, err) {
-		t.FailNow()
-	}
-
-	assertHasAnnotation(t, out.String(), annos)
+	filter.FsSlice = annosFs
+	assertHasAnnotation(t, filtertest_test.RunFilter(t, input, filter), annos)
```

Comment:
if you add the format filter then you don't need the new assertion -- you can compare the full expected / actual text

Reply:
excellent suggestion.